lunatrace/bsl/hasura/metadata/databases/lunatrace/tables/public_default_branch_builds.yaml

```diff
@@ -0,0 +1,105 @@
+table:
+  schema: public
+  name: default_branch_builds
+object_relationships:
+  - name: project
+    using:
+      manual_configuration:
+        remote_table:
+          schema: public
+          name: projects
+        insertion_order: null
+        column_mapping:
+          project_id: id
+array_relationships:
+  - name: findings
+    using:
+      manual_configuration:
+        remote_table:
+          schema: public
+          name: findings
+        insertion_order: null
+        column_mapping:
+          id: build_id
+  - name: scans
+    using:
+      manual_configuration:
+        remote_table:
+          schema: public
+          name: scans
+        insertion_order: null
+        column_mapping:
+          id: build_id
+insert_permissions:
+  - role: service
+    permission:
+      check: {}
+      columns:
+        - agent_access_token
```

Comment:
We should strip this column unless it's actually needed (since it's the only sensitive value in this table)